pkg/log/logger_ctl.go

```diff
@@ -139,19 +139,19 @@ func (c *ctlHandler) Handle(r slog.Record) error {
 	return err
 }
 
-type colour struct {
+type color struct {
 	renderer string
 	width    int
 }
 
-func newColour(c ctc.Color, msg string) colour {
-	return colour{
+func newColour(c ctc.Color, msg string) color {
+	return color{
 		renderer: fmt.Sprintf("%s%s%s", c, msg, ctc.Reset),
 		width:    stringWidth(msg),
 	}
 }
 
-var levelColour = map[string]colour{
+var levelColour = map[string]color{
```

Comment:
```suggestion
var levelColor = map[string]color{
```

Reply:
good, why the misspell not found that....😂😂